cmd/netgoal/generate.go

```diff
@@ -125,13 +143,13 @@ template modes for -t:`,
 				reportErrorf("must specify number of relays with -R")
 			}
 
-			err = generateNetworkTemplate(outputFilename, walletsToGenerate, relaysToGenerate, nodeHostsToGenerate, nodesToGenerate, baseNode, baseRelay)
+			err = generateNetworkTemplate(outputFilename, walletsToGenerate, relaysToGenerate, nodeHostsToGenerate, nodesToGenerate, nonPartnodesHostsToGenerate, baseNode, baseNonParticipatingNode, baseRelay)
```

Comment:
at some point this growing parameter list might look better as a parameter object. maybe next time.